test/evergreen.yml

```diff
@@ -190,6 +190,15 @@ functions:
         set -o errexit
         set -o verbose
         ./t ${checkpoint_args} 2>&1
+  
+  "format test script":
+    command: shell.exec
+    params:
+      working_dir: "wiredtiger/test/format"
+      script: |
+        set -o errexit
+        set -o verbose
+        ${test_env_vars|} ./format.sh ${format_test_script_args|} ${smp_command|} 2>&1
```

Comment:
Can this section being moved up next to the existing `"test format"` function? And to keep the naming consistent the older function is better to be called `"format test"` instead :)

Reply:
Good idea. I will move and rename functions.